### DISCUSSION TITLE
Is downgrading completely patched?

### DISCUSSION THREAD

**Discussion by TheAnimemer** [3kh0/ext-remover#710] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was trying to downgrade my chromebook to use some other exploits, and I got a message saying that essentially my recovery device is outdated and it doesnt let me recover using it. I saw a thread asking something similar and it was answered saying that downgrading is patched. I'm curious if there are any other ways to downgrade or if I should give up on that idea. 

Also I saw people warning that you could get in serious trouble for downgrading, and while that risk is obvious, would administration know i downgraded without seeing my chromebook? Sorry if this has been asked before but there are many threads, most of which lack answers, and after an hour of looking through them I'm deciding to start a thread instead.

**Comment by AshtonDavies** [comment]
There might be hopes if Mercury Workshop looks into a way to glitch around it.

What I know is:
- RMA shims still work if they happen to be able to help with this, I don't know
- Halcyon may or may not be patched because of this patch, the recovery screen might launch it before checking the version
- Outdated recovery images were patched, but...
- It might be possible for a tool to modify an outdated image as a fake "update" or "reflash" to bypass this

**Comment by ghost** [comment, marked as answer]
you can try to use https://fog.gay/ , if you do that i think you can downgrade

**Reply by TheAnimemer** [reply]
This looks like it still works, but it seems like downgrading in general unenrolls your chromebook. I won't be downgrading since I don't want administration to know that I am fucking with their computers. I also was hoping that you could reverse the downgrade for when i give my chromebook back at the end of the year, but I can't find info on that. Thanks for your help!

**Reply by AshtonDavies** [reply]
> This looks like it still works, but it seems like downgrading in general unenrolls your chromebook. I won't be downgrading since I don't want administration to know that I am fucking with their computers. I also was hoping that you could reverse the downgrade for when i give my chromebook back at the end of the year, but I can't find info on that. Thanks for your help!

No downgrading doesn't unenroll the Chromebook, and both are patched, unfortunately.
